backend/plugins/posclient_api/src/modules/posclient/graphql/resolvers/customResolvers/posConfig.ts

```
@@ -0,0 +1,15 @@
+import { IContext } from '~/connectionResolvers';
+import { IConfigDocument } from '~/modules/posclient/@types/configs';
+
+export default {
+  async slots(config: IConfigDocument, _args, { models }: IContext) {
+    return await models.PosSlots.find({ posId: config.posId });
+  },
+
+  allowTypes(config: IConfigDocument, _args, {}: IContext) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)